awx/ui_next/src/screens/Template/WorkflowJobTemplateVisualizer/Visualizer.jsx

```diff
@@ -405,16 +405,7 @@ function Visualizer({ template, i18n }) {
                   failure_nodes: [],
                   always_nodes: [],
                 };
-                if (node.promptValues?.removedCredentials?.length > 0) {
```

Comment:
When writing the tests, I realized this code was specific to editing a node and isn't needed in this block.  It was probably just copied over.  We only need to remove _overriding_ credentials.  Since this is a new node, there are no overriding credentials to remove (only overriding credentials to add)